lib/mix/tasks/expo.msguniq.ex

```diff
@@ -0,0 +1,102 @@
+defmodule Mix.Tasks.Expo.Msguniq do
+  @shortdoc "Unify duplicate translations in message catalog"
+
+  @moduledoc """
+  Unifies duplicate translations in the given PO file.
+
+  By default, this task outputs the file on standard output. If you want to
+  *overwrite* the given PO file, pass in the `--output` flag.
+
+  ## Usage
+
+      mix expo.msguniq PO_FILE [--output=OUTPUT_FILE]
+
+  ## Options
+
+  * `--output` (`-o`) - Default: `-` - File to store the output at. `-` for
+    STDOUT.
+
+  """
+  @moduledoc since: "0.5.0"
+
+  use Mix.Task
+
+  alias Expo.Message
+  alias Expo.Messages
+  alias Expo.PO
+  alias Expo.PO.DuplicateMessagesError
+
+  @switches [
+    output: :string
```

Comment:
Should be `--output-file`

Reply:
Let's go with `--output-file`, sounds good.